build.cake

```diff
@@ -0,0 +1,60 @@
+#tool nuget:?package=NUnit.ConsoleRunner&version=3.4.0
+
+var target = Argument("target", "Default");
+var configuration = Argument("configuration", "Release");
+
+
+var buildDir = Directory($"./build/{configuration}/");
+var solutionFile = "./src/Grok.Net.sln";
+var outputNupkg = $"./src/Grok.Net/bin/{configuration}/*.nupkg";
+
+Task("Clean")
+    .Does(() =>
+{
+    CleanDirectory(buildDir);
+});
+
+Task("Restore-NuGet-Packages")
+    .IsDependentOn("Clean")
+    .Does(() =>
+{
+    NuGetRestore(solutionFile);
```

Comment:
NuGetRestore uses the nuget cli instead of dotnet. Apologies I've only really written cake scripts for full framework projects before so I used what I know! I've updated to use DotNetRestore